### ISSUE TITLE
list_size in serving/prediction

### ISSUE THREAD

**Issue by MiladShahidi** [tensorflow/ranking#108]
I have successfully trained and exported a model and deployed it on GCP for testing. I'm also able to send a query-document (context-example) pair and get a prediction back.
I'm working on a recommender system, and need to score 1000s of items for a given "query" (user in my case). But I'm not sure how to do that?
Say I want to get recommendations for a given user. I understand that the list_size is fixed (say 100) during training. But is it also fixed during serving/prediction? Can I send a prediction request to my deployed model with a query and a list of 10,000 documents (e.g. in example-in-example format) and get a list of 10,000 scores back?
If so, I'm wondering how that is done, given that the number of logits calculated by the model is fixed at list_size.
